src/clone/Cloner.php

```diff
@@ -73,7 +73,7 @@ public function validate_ee_version() : void {
 		$matches['version'] = preg_replace( '/-nightly.*$/', '', $matches['version'] );
 
 		if ( Comparator::lessThan( $matches['version'], '4.1.3' ) ) {
-			throw new \Exception( 'EasyEngine version on \'' . $this->host . '\' is \'' . $matches['version'] . '\' which is less than minimum required version \'4.1.3\' for cloning site.' );
+			EE::error('EasyEngine version on \'' . $this->host . '\' is \'' . $matches['version'] . '\' which is less than minimum required version \'4.1.3\' for cloning site.');
```

Comment:
Please follow proper spacing conventions as per the rest of the codebase

Reply:
Fixed in https://github.com/EasyEngine/site-command/pull/357/commits/a63b100199790ba20194b57906354d423353057e